packages/perseus-editor/src/widgets/interactive-graph-editor.tsx

```diff
@@ -223,6 +229,7 @@ class InteractiveGraphEditor extends React.Component<Props> {
                 markings: this.props.markings,
                 showProtractor: this.props.showProtractor,
                 showRuler: this.props.showRuler,
+                showTooltips: this.props.showTooltips,
```

Comment:
Doesn't do anything, but we'll want this here in the future if we want it to eventually do something.